### PR TITLE
Add a simple README.md file

### PR DESCRIPTION
**Pull request by vivekkumar611**
Updated the README.md file.

### CODE REVIEW

**Comment by msingh219** [comment]
This README is too short.It would be nice to see an example.

**Review by msingh219 — COMMENTED** [review]
Period is missing.

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# rearrange
\ No newline at end of file
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName, FirstName" into "FirstName LastName"
+
+# Example
```
Comment (msingh219) [inline README.md]: Please use ## to make it smaller.

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+Calling 'rerrange_name("Turing Alan")' will return "Alan Turing"
```
Comment (msingh219) [inline README.md]: Add one more example

**Review by msingh219 — COMMENTED** [review]
Please see comments

**Comment by vivekkumar611** [comment]
All comments resolved. Please take another look.

**Pull request merged**